hydrolib/core/dflowfm/mdu/models.py

```diff
@@ -553,6 +685,10 @@ class Comments(INIBasedModel.Comments):
         computedairdensity: Optional[str] = Field(
             "Compute air density yes/no (), 1/0, default 0.", alias="computedAirdensity"
         )
+        stresstowind: Optional[str] = Field(
+            "Switch between Wind speed (=0) and wind stress (=1) approach for wind forcing.",
```

Comment:
is this text correct? it sounds like a bool in this case, or am I interperting this incorrectly?

Reply:
According to the manual's default value, it's a float/double. Reading the description, it made more sense to make it an int with only two options: 0 or 1.